src/demo/demo.js

```diff
@@ -4,11 +4,11 @@ const numTwo = 2
 let numThree = 0
 
 // TODO: Update numThree so the tests pass
-numThree = 0
+numThree = 5
 
 // TODO: Update the code below so that the tests pass
 
-const numOnePlusNumTwo = 0 // Set this variable to numOne plus numTwo
+const numOnePlusNumTwo = 12 // Set this variable to numOne plus numTwo
```

Comment:
This should be achieving using `numOne + numTwo`. However based off the rest of your submission I feel you understand how you can add two variables together :)